src/sp_repo_review/checks/pyproject.py

```diff
@@ -241,5 +241,65 @@ def check(pyproject: dict[str, Any]) -> bool:
         return "filterwarnings" in options
 
 
+class PP310(PyProject):
+    "Tests target is test not test (spec13)"
+
+    requires = {"PP301"}
+    url = mk_url("pytest")
+
+    @staticmethod
+    def check(pyproject: dict[str, Any]) -> bool | None:
+        """
+
+        Tests target should be `tests` not `test`
+
+        ```toml
+        [project.optional-dependencies]
+        tests = [
+            'pytest',
+            ...
+        ]
+        ```
+        """
+        if "tool" not in pyproject:
+            return None
+        if "project.optional-dependencies" not in pyproject["tool"]:
+            return None
+        optional_deps = pyproject["tool"]["project.optional-dependencies"]
+        if "tests" in optional_deps:
+            return True
+        return "test" not in optional_deps
+
+
+class PP311(PyProject):
+    "Tests target is `docs not` `doc` (spec13)"
+
+    requires = {"PP301"}
+    url = mk_url("pytest")
+
+    @staticmethod
+    def check(pyproject: dict[str, Any]) -> bool | None:
+        """
+
+        docs target should be `docs` not `doc`
```

Comment:
```suggestion
        docs extra should be `docs` not `doc`
```

Reply:
Also, I am not interested in having a check that goes against the official Python packaging specifications, which state `test` and `doc` are reserved for this. This means tools (like Hatch) can assume special meanings for these extras. IMO, SPECs are only for standardizing things _not_ already standardized across Python, and so we can't enforce a SPEC over a the official core specification.

https://packaging.python.org/en/latest/specifications/core-metadata/#provides-extra-multiple-use

However, given that `[docs]` is more popular, and I can't see any PEP source for that recommendation (other than PEPs stating it is the source of truth), I think we could see if it can be changed or removed, in which case such a check is fine.